src/components/agents/Agent.jsx

```diff
@@ -41,7 +48,7 @@ export default class Agent extends Component {
       <div>
         <b>Hosts: </b>
         {agent.hosts.map((h, index) => (
-          <span key={index + h.species}><i>{h.genus} {h.species}{h.subSpecies ? ' ' : ''}{h.subSpecies}</i>{index < agent.hosts.length - 1 ? ', ' : ''}</span>
+          <span key={index + h.species}><a style={{ cursor: 'pointer' }} onClick={this.createInteractionRoute}><i data-interaction={h.interactionId}>{h.genus} {h.species}{h.subSpecies ? ' ' : ''}{h.subSpecies}</i></a>{index < agent.hosts.length - 1 ? ', ' : ''}</span>
```

Comment:
Need to add a linting rule for this, but can we break this into multiple lines to help with the wrapping?